README.md

```diff
@@ -1,5 +1,9 @@
+# Warning
+|:warning: This fork is for _prototyping purpose only_. If you want to submit a PR to the Open Hospital project starts from the [official ISF (Informatici Senza Frontiere) repos](https://github.com/informatici).      |
+|:---------------------------|
+
 # Open Hospital - Core
-[![Java CI](https://github.com/informatici/openhospital-core/workflows/Java%20CI%20with%20Maven/badge.svg)](https://github.com/informatici/openhospital-core/actions?query=workflow%3A%22Java+CI+with+Maven%22)
+[![Java CI](https://github.com/intesys/openhospital-core/workflows/Java%20CI%20with%20Maven/badge.svg)](https://github.com/intesys/openhospital-core/actions?query=workflow%3A%22Java+CI+with+Maven%22)
```

Comment:
Do not change repos; this must run against the Informatici repo and not your local one